src/main/scala/io/viash/engines/DockerEngine.scala

```diff
@@ -215,12 +223,17 @@ final case class DockerEngine(
     target_registry.orElse(config.links.docker_registry)
   }
 
+  def getTargetPackageNameWithFallback(config: Config): Option[String] = {
+    target_package.orElse(config.package_config.flatMap(_.name)).filter(_.nonEmpty)
+  }
+
   def getTargetIdentifier(config: Config): String = {
     val targetImageInfo = Docker.getImageInfo(
       config = Some(config),
       engineId = Some(id),
       registry = getTargetRegistryWithFallback(config),
       organization = target_organization,
```

Comment:
Shouldn't this use a fallback value too? Currently this doesn't use a fallback value and it seems odd to me.

Reply:
Not only that, I would suggest to add a placeholder/implicit organisation: `-` or `viash-hub` that can be overridden using `target_organisation` if need be.

Reply:
The last few days I've been wondering ... would it make sense to add **another** config file, this time a global one.
Order of preference would then be `component config` > `package config` > `global config`.

Also, can someone explain me again why we have `organization` and `target_organization`, or `registry` and `target_registry`, ... ? I understand it's to be able to override the built container info, but why is there no fallback to the "non target_*" value?